src/CssManager.php

```diff
@@ -950,7 +950,7 @@ function setCSSboxshadow($v)
 					}
 				}
 			}
-			if (!$new['col']) {
+			if (!isset($new['col'])) {
```

Comment:
This one actually has a different meaning. when `$new['col']` is an empty string, it is set, but would pass in the updated case... Using `empty` or `!empty` would be better.